### PR TITLE
verify xgboost with SHAP

### PR DESCRIPTION
**Pull request by Yancey0623**
This PR verified xgboost model and explain it with the ANALYZE statement.
a part work of #749

### CODE REVIEW

File: sql/codegen_xgboost_test.go [hunk 1]
```diff
@@ -32,6 +32,11 @@ COLUMN sepal_length, sepal_width, petal_length, petal_width
 LABEL class 
 INTO sqlflow_models.my_xgboost_model;
 `
+const testAnalyzeTreeModelSelectIris = `
+SELECT * FROM iris.train
+ANALYZE sqlflow_models.my_xgboost_model
```
Comment (weiguoz) [inline sql/codegen_xgboost_test.go:37]: sqlflow_models.my_xgboost_model -> XGB.my_xgboost_model ?

https://github.com/Yancey1989/sqlflow/blob/497ca9691a006652a4a7b6b0e8e46267d69543c4/sql/codegen_analyze.go#L88-L90

Comment (Yancey0623) [reply sql/codegen_xgboost_test.go:37]: `XGB.` is the prefix of Estimator, I think the model name can be any user-defined name.

Comment (weiguoz) [reply sql/codegen_xgboost_test.go:37]: Sorry, I misunderstood the concepts of estimator and model.

**Review by weiguoz — APPROVED** [review]
LGTM

File: sql/codegen_xgboost_test.go [hunk 2]
```diff
@@ -32,6 +32,11 @@ COLUMN sepal_length, sepal_width, petal_length, petal_width
...
+USING TreeExplainer;
+	`
```
Comment (typhoonzero) [inline sql/codegen_xgboost_test.go:39]: remove this tab?

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**